### PR TITLE
Created download folder if not exists

### PR DESCRIPTION
**Pull request by sujata-m**
2 things got fixed in this PR - 

1. created the `downloads` folder if not exists
2. Fixed Lock expired error by putitng self.format function in a thread

### CODE REVIEW

**Comment by susrisha** [comment]
Make sure it addresses the message lock problem as well.

**Comment by susrisha** [comment]
In `osw_formatter.py` add the following code
```python
# at the top
import threading
# at the method
self.format(upload_message)
process_thread = threading.Thread(target=self.format,args=[upload_message])
process_thread.start()
```

File: src/formatter.py [hunk 1]
```diff
@@ -34,7 +37,7 @@ def format(self):
                 logger.info(f' Downloaded file path: {downloaded_file_path}')
                 f = Formatter(workdir=DOWNLOAD_FILE_PATH, file_path=downloaded_file_path, prefix=self.prefix)
                 formatter = f.osw2osm()
-                OSWFormat.clean_up(downloaded_file_path)
+                # OSWFormat.clean_up(downloaded_file_path)
```
Comment (susrisha) [inline src/formatter.py:40]: why is clean up removed?

Comment (sujata-m) [reply src/formatter.py:40]: now self.format is in Thread (async in nature)and we have same file name for all the events - 
Thread 1 - calls the `self.format` 
Thread 2 - calls the `self.format` 

What happens- when Thread 2 is processing the file, Thread 1 deletes that

**Review by susrisha — APPROVED** [review]

**Pull request merged**